x-pack/plugins/apm/server/lib/transaction_groups/get_coldstart_rate.ts

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  FAAS_COLDSTART,
+  SERVICE_NAME,
+  TRANSACTION_NAME,
+  TRANSACTION_TYPE,
+} from '../../../common/elasticsearch_fieldnames';
+import { offsetPreviousPeriodCoordinates } from '../../../common/utils/offset_previous_period_coordinate';
+import { kqlQuery, rangeQuery } from '../../../../observability/server';
+import { environmentQuery } from '../../../common/utils/environment_query';
+import { Coordinate } from '../../../typings/timeseries';
+import {
+  getDocumentTypeFilterForTransactions,
+  getProcessorEventForTransactions,
+} from '../helpers/transactions';
+import { getBucketSizeForAggregatedTransactions } from '../helpers/get_bucket_size_for_aggregated_transactions';
+import { Setup } from '../helpers/setup_request';
+import {
+  calculateTransactionColdstartRate,
+  getColdstartAggregation,
+  getTransactionColdstartRateTimeSeries,
+} from '../helpers/transaction_coldstart_rate';
+
+export async function getColdstartRate({
+  environment,
+  kuery,
+  serviceName,
+  transactionType,
+  transactionName,
+  setup,
+  searchAggregatedTransactions,
+  start,
+  end,
+}: {
+  environment: string;
+  kuery: string;
+  serviceName: string;
+  transactionType?: string;
+  transactionName?: string;
+  setup: Setup;
+  searchAggregatedTransactions: boolean;
+  start: number;
+  end: number;
+}): Promise<{
+  noHits: boolean;
```

Comment:
We've recently removed all instances of `noHits` because it seemed liked an anti-pattern. In this case it doesn't looks like it's used. Can we remove?